pkg/scheduler/replica/replication_group.go

```diff
@@ -239,3 +244,46 @@ func (g *replicationGroup[T, R]) GetTaskSizePerNode() map[node.ID]int {
 	}
 	return res
 }
+
+type Map[T ReplicationID, R Replication[T]] struct {
```

Comment:
It seems that this Map is only used in this file. We can make it private by changing its name to rMap.